lgc/test/PeepholeOptLog2Underflow.lgc

```diff
@@ -0,0 +1,47 @@
+; Test that log2 underflow transform is correctly applied.
+
+; RUN: lgc -mcpu=gfx1010 -print-after=lgc-patch-peephole-opt -o - - <%s 2>&1 | FileCheck --check-prefixes=CHECK %s
```

Comment:
I think this test can just use the -passes=lgc-patch-peephole-opt and update_test_checks. And moving it to lgc/test/Transforms/PeepholeOpt for consistency would be grand.